src/components/UserProfile/TeamsAndProjects/TeamsAndProjects.css

```diff
@@ -3,13 +3,36 @@
   background-color: #fafafa;
   margin-top: 20px;
 }
+
+.projecttable-container .col-md-2,
+.teamtable-container .col-md-2 {
+  padding: 0px;
+}
+
+.container {
+  padding: 0;
+}
+
+.row {
+  margin-left: 0;
+}
+
+.teamtable-container .container,
+.projecttable-container .container {
+  padding: 0px;
+}
+
+.teamtable-container .row,
+.projecttable-container .row {
+  margin: 0px;
+}
+
 .btn-addteam,
 .btn-addproject {
```

Comment:
```suggestion
.btn-addproject {
  width: 100%;
```
Before:
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/23086054/149971a7-3a12-46bc-a201-bd66adf0372d)
After:
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/23086054/62e3db37-3ee3-43b4-8aea-c96e2b9e1a8a)